### PR TITLE
feat(pool): calculated price ratio when adding to pool

### PR DESCRIPTION
**Pull request by ngrujicic**

### CODE REVIEW

File: src/app/types/index.ts [hunk 1]
```diff
@@ -15,3 +15,7 @@ export type PoolsCardsProps = {
     assetTokenIcon: string;
   };
 };
+
+export type InputEditedProps = {
+  inputType: "exactIn" | "exactOut";
```
Comment (markoMVP85) [inline src/app/types/index.ts]: "exactIn" | "exactOut" can be an enum rather than a literal enum

File: src/app/util/helper.ts [hunk 2]
```diff
@@ -17,9 +17,8 @@ export const reduceAddress = (address: string | undefined, lengthLeft: number, l
   return t("wallet.notConnected");
 };
 
-export const calculateSlippage = (base: string, percent: number | undefined = 0) => {
-  const result = (parseFloat(base) * percent) / 100;
-  return Math.ceil(result);
+export const calculateSlippage = (tokenValue: number, slippageValue: number) => {
+  return tokenValue - (tokenValue * slippageValue) / 100;
```
Comment (markoMVP85) [inline src/app/util/helper.ts]: Better use Decimal.js if u have installed it

File: src/components/organism/SwapTokens/index.tsx [hunk 3]
```diff
@@ -149,39 +148,39 @@ const SwapTokens = () => {
...
             : selectedTokens.tokenB.decimals
         );
 
-        const assetTokenWithSlippage =
+        const nativeTokenWithSlippage =
           inputEdited.inputType === InputEditedType.exactIn
-            ? assetTokenNoDecimals - (assetTokenNoDecimals * slippageValue) / 100
-            : assetTokenNoDecimals + (assetTokenNoDecimals * slippageValue) / 100;
+            ? nativeTokenNoDecimals - (nativeTokenNoDecimals * slippageValue) / 100
+            : nativeTokenNoDecimals + (nativeTokenNoDecimals * slippageValue) / 100;
```
Comment (markoMVP85) [inline src/components/organism/SwapTokens/index.tsx]: Same here. Essentially wherever u have some kind of calculation, use decimal package

File: src/app/types/index.ts (hunk 1 above)
Comment (ngrujicic) [reply src/app/types/index.ts]: i am not sure i understand, did you mean i should do something like this below ?
`export type InputEditedProps = {
  inputType: InputEditedType.exactIn | InputEditedType.exactOut;
};`

Comment (markoMVP85) [reply src/app/types/index.ts]: no. U have "exactIn" | "exactOut" that should be enum EnumName {EXACT_IN = "exactIn" | EXACT_OUT = "exactOut"}
inputType: EnumName

**Review by markoMVP85 — APPROVED** [review]

**Pull request merged**